pages/home.md

```diff
@@ -58,6 +58,10 @@ graphic_list:
   <div class="grid-container">
     <h2 class="font-lang-xl margin-y-0 line-height-serif-2">Showcase</h2>
     <div class="grid-row grid-gap tablet:margin-top-3">
+      <div class="tablet:grid-col">
+        <img src="{{ site.baseurl }}/img/webp/home/showcase/NIH.webp" alt="Preview image of National Institutes of Health Library" width="960" height="540">
```

Comment:
To match the other images, the alt text should mention that it is an image of the website.
```suggestion
        <img src="{{ site.baseurl }}/img/webp/home/showcase/NIH.webp" alt="Preview image of National Institutes of Health Library home page" width="960" height="540">
```